src/PlacePicker/components/PlacePickerItem.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+import * as React from 'react';
+import { RowOption } from '../../RowOption';
+import type { PlaceType } from '../PlacePickerTypes';
+
+type Props = {|
+  +place: PlaceType,
+  +onItemPress: (place: PlaceType) => void,
+  +onAddPress: (place: PlaceType) => void,
+|};
+
+function createPressHandler(func, place) {
```

Comment:
Could you move it to helpers maybe?

Reply:
This is the helper just for the PlacePickerItem so I don't think we would like to use it anywhere else. So I don't see the reason why put this to helpers if it is and will be used it just inside PlacePicker.